### PR TITLE
Split sqlalchemy_utc module into a package

### PR DESCRIPTION
**Pull request by gwax**
This PR breaks the `sqlalchemy_utc.py` module into a `sqlalchemy_utc/` package. The API is undisturbed. This PR doe not introduce any functional changes.

### CODE REVIEW

**Comment by gwax** [comment]
@dahlia this PR is now ready.

File: .travis.yml [hunk 1]
```diff
@@ -34,26 +38,29 @@ before_install:
...
+
 install:
 - if [[ "$PIP_OPTS" != "" ]]; then
     pip install $PIP_OPTS "SQLAlchemy $SQLALCHEMY_VER";
   else
     pip install "SQLAlchemy $SQLALCHEMY_VER";
   fi
 - pip install -e .
-- pip install pytest flake8 "import-order >= 0.0.10"
+- pip install pytest flake8 flake8-import-order
```
Comment (dahlia) [inline .travis.yml:49]: This change should be reflected to tox.ini as well (the duplication per se is problematic though).

**Review by dahlia — APPROVED** [review]

File: .travis.yml (hunk 1 above)
Comment (gwax) [reply .travis.yml:49]: I'll see what I can do.

Comment (gwax) [reply .travis.yml:49]: `tox.ini` is up to date now.

I will come back to unifying tox and travis in a later PR.

**Review by dahlia — APPROVED** [review]

**Comment by dahlia** [comment]
Thanks!

**Pull request merged**